packages/wallet/src/wallet.ts

```diff
@@ -320,12 +320,19 @@ export class Wallet {
    * @throws Error if the address is not found or decryption fails
    */
   async getPrivateKey(addressPath: string, password: string): Promise<string> {
-    const mnemonic = await this.getMnemonic(password);
-    const hdWallet = this.core.HDWallet.createWithMnemonic(mnemonic, '');
+    try {
+      const mnemonic = await this.getMnemonic(password);
+      const hdWallet = this.core.HDWallet.createWithMnemonic(mnemonic, '');
+
+      const privateKey = hdWallet.getKey(this.core.CoinType.pactus, addressPath);
+      const prefix = this.privateKeyPrefix();
 
-    const privateKey = hdWallet.getKey(this.core.CoinType.pactus, addressPath);
+      const privateKeyStr = encodeBech32WithType(prefix, privateKey.data(), 3);
```

Comment:
Please define constant for Signature Algorithm Type, like 

https://github.com/pactus-project/pactus/blob/d925a7c2d61be255872447a455d8ca0b6d08a69b/crypto/address.go#L14-L18